mandatory/2-function-creation.js

```diff
@@ -25,7 +37,27 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
+  for (let i=0; i< arrayOfStrings.length; i++) {
+    arrayOfStrings[i]= arrayOfStrings[0].replace("/",'');
```

Comment:
Hi Timea, i really like the layout of code, very easy to read. However, if you are able to simplify the logic here by deleting some  lines of code. 
like arrayOfStrings[i]= arrayOfStrings[0].replace("/",'').toLowerCase().trim();